packages/@lwc/engine-core/src/framework/upgradable-element.ts

```diff
@@ -42,6 +44,21 @@ export function getUpgradableConstructor(
             }
         }
     };
+    if (features.ENABLE_NATIVE_CUSTOM_ELEMENT_LIFECYCLE) {
+        CE.prototype.connectedCallback = function () {
+            // FIXME: why is this undefined sometimes?
```

Comment:
it could be undefined if the constructor of the LWC Component throws during construction, in which case the instance `this` will never get the associated VM attached to it.